src/styles/blocks/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  background: #fff;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+
+  margin: 0 3px 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+
+  margin: 0 0 4px;
+}
+
+.card__product-code {
+  display: flex;
+
+  line-height: 14px;
+
+  color: #616070;
+
+  margin: 0 0 16px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
```

Comment:
not sure why flex-end - generally doesn't really matter in this case
yet center is a better option for centered elements

Reply:
changes are done